help/en/html/doc/Technical/JVMCapabilities.shtml

```diff
@@ -522,7 +520,19 @@
           </tr>
 
           <tr>
-            <td>4.2</td>
+            <td>5.*</td>
+            <td>11</td>
+            <td>10.10</td>
+            <td>Windows Vista or later<br>64 bit only</td>
+            <td>5.5+</td>
+            <!-- not sure what 5.5 means here, since JMRI 4.x works on CentOS 7, which has Linux kernel 3.10 -->
```

Comment:
Unless we know what the `Linux` column means, I suggest that we remove it. The current mainline kernel is 5.15 so this column cannot be about the kernel. I thought it could be Debian, but there doesn't seem to have been a Debian 5.5.
https://www.kernel.org/
https://en.wikipedia.org/wiki/Debian_version_history#Debian_5.0_(Lenny)

Reply:
5.5 appears to be Redhat

Reply:
It may also refer to Oracle Linux.
https://www.java.com/en/download/help/sysreq.html

Reply:
@bobjacobsen 
Unless anyone has anything more to add, I think @PUlvestad is right about 5.5 refers to Oracle Linux as noted in this document:
https://www.java.com/en/download/help/sysreq.html

So I suggest you change "Linux" to "Oracle Linux" and then merge this PR.